MetaTags/Services/MetaTagsService.cs

```diff
@@ -124,13 +124,13 @@ private string GetHostUrl()
 
         public string GetMediaUrl(string path)
         {
-            if (string.IsNullOrWhiteSpace(path))
-            {
-                return string.Empty;
+            if (string.IsNullOrWhiteSpace(path))
+            {
+                return string.Empty;
             }
 
             var imageUrl = _mediaFileStore.MapPathToPublicUrl(path);
-            return imageUrl.StartsWith("http") ? imageUrl : $"{GetHostUrl()}{imageUrl}";
+            return imageUrl.StartsWith("http") ? imageUrl : $"{GetHostUrl()}{imageUrl}?width=1200&height=630&rmode=crop";
```

Comment:
We'll need to make sure that `630` is added to the app settings in the site boilerplate.

https://github.com/EtchUK/Etch.OrchardCore.SiteBoilerplate/blob/main/Content/src/Etch.OrchardCore.SiteBoilerplate/appsettings.json

Reply:
Also not a fan of 1200 & 630 being magic numbers, can we move them to constants.

Reply:
Committed some changes. Is that what you meant?

Reply:
Not quite. You need to add the following to the top of the class.

```
private const int ImageCropHeight = 630;
private const int ImageCropWidth = 1200;
```

Then use those variables 👍